CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+7.10.7
+======
+
+*   (improvement) Also add `Router` under interface class name in the service locator in the `BaseController`.
```

Comment:
Will be fixed locally

Reply:
https://github.com/Becklyn/RadBundle/commit/127cfb13bc472f4da39470fa8812aaaa95d79542